css/customize-snapshots.css

```diff
@@ -216,4 +216,122 @@
 #customize-snapshot .snapshot-controls{
 	list-style: none;
 	margin: 0;
-}
\ No newline at end of file
+}
+
+.snapshot-future-date-notification.notice{
```

Comment:
There should be a space between all the classes and the opening bracket. Which goes for the statements below, as well.